deepfence_frontend/apps/dashboard/src/utils/registry.ts

```diff
@@ -0,0 +1,17 @@
+import { isNil } from 'lodash-es';
+
+import { ModelRegistryListResp } from '@/api/generated';
+
+export function getRegistryDisplayId(registry: ModelRegistryListResp): string {
+  if (
+    registry.registry_type === 'docker_hub' &&
```

Comment:
Any reason why just docker_hub?

Reply:
that's what we have currently, we will add more as api support is added.